<launch>
<arg name="use_ping" default="false"/>
<arg name="id" default="5"/>

<arg name="device_name" default="/dev/ttyACM0"/>
<arg name="baud_rate" default="1000000"/>
<arg name="protocol_version" default="1.0"/>

<param name="ping" value="$(arg use_ping)" type="bool"/>
<param name="ping_id" value="$(arg id)"/>

<param name="device_name" value="$(arg device_name)"/>
<param name="baud_rate" value="$(arg baud_rate)"/>
<param name="protocol_version" value="$(arg protocol_version)"/>

<node name="position_control" pkg="dynamixel_workbench_controllers"
type="position_control" output="screen"/>
</launch>
